core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -54,10 +55,11 @@ class ZooKeeperQuorumImplementation(val zookeeper: EmbeddedZookeeper,
                                     val zkClient: KafkaZkClient,
                                     val adminZkClient: AdminZkClient,
                                     val log: Logging) extends QuorumImplementation {
-  override def createAndStartBroker(config: KafkaConfig,
-                                    time: Time): KafkaBroker = {
+  override def createAndMaybeStartBroker(config: KafkaConfig,
```

Comment:
Let's just call this `createBroker` and have the startup argument speak for itself...